src/Client.ts

```diff
@@ -96,6 +98,7 @@ export function createClient({
         newsroomDomains: new NewsroomDomains.Client(apiClient),
         newsroomGalleries: new NewsroomGalleries.Client(apiClient),
         newsroomHub: new NewsroomHub.Client(apiClient),
+        pricingTables: new PricingTables.Client(apiClient),
```

Comment:
> After I open the endpoint, will it require any changes here in SDK?

Actually, I'm not sure about the best approach here, since `createClient` will require an `accessToken` and it is required:
https://github.com/prezly/javascript-sdk/blob/028170fc03159c90736382fa062618fe96d3cdbe/src/api/ApiClient.ts#L8

- Maybe we should also add a new client creator that doesn't require `accessToken`? Something like `createPublicClient`?
- Or make the `accessToken` optional?

@e1himself @aspirisen @fgyimah what do you suggest here?

Reply:
We could pass in a fixed access token just as we do for [prezly-to-contentful](https://github.com/prezly/prezly-to-contentful) and that could work, @marko-ogg can give more details if that can help.

See example [here](https://github.com/prezly/prezly-to-contentful/blob/635056039e526b2c19f0df48f2e923305c5da8a5/pages/api/newsrooms.ts#L11)

Reply:
I just realized we have another public API (`register`) that doesn't require an `accessToken`, I looked into app for inspiration and I believe we won't need to change anything here in SDK.

Achievable by overriding `header` when creating client:

```ts
createPrezlyClient({
    accessToken: '',
    headers: { authorization: '' },
});
```